dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -949,7 +949,8 @@ public ExperimentResults getResults(final Experiment experiment, final User user
     }
 
     @CloseDBIfOpened
-    private List<Experiment> cacheRunningExperiments() throws DotDataException {
+    @Override
+    public List<Experiment> cacheRunningExperiments() throws DotDataException {
```

Comment:
I think this would be nicer if you do something such as:
```
 @CloseDBIfOpened
    @Override
    public List<Experiment> populateCacheExperimentsByState(final String cacheKey, final Status status) throws DotDataException {
        final List<Experiment> experiments = FactoryLocator
            .getExperimentsFactory()
            .list(ExperimentFilter.builder().statuses(set(status)).build());
        experimentsCache.putList(cacheKey, experiments);
        return experiments;
    }
```

Reply:
Not new. This was implemented by Vico some time ago. @jdotcms